### PR TITLE
Add new file demo

### PR DESCRIPTION
**Pull request by foxhsiao**
@chriszTW 功能完成了可以協助我確認一下

- [ ] function 1
- [ ] function 2
- [ ] function 3

@Lyndon1983 help review this

### CODE REVIEW

File: demo.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# test
```
Comment (foxhsiao) [inline demo.md]: @philifalo 寫得跟屎一樣..

Comment (foxhsiao) [inline demo.md]: #4 已解決這個問題

**Comment by Lyndon1983** [comment]
@foxhsiao got it.

**Comment by chriszTW** [comment]
check ok!

**Pull request merged**